src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,12 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a block in a Person's address in the address book.
+ * Guarantees: immutable
+ */
+public class Block {
+    public final String value;
```

Comment:
Try to make it private. 
In general, start off with private members, then use protected or public only if absolutely necessary.